nautobot/ipam/views.py

```diff
@@ -1414,15 +1404,28 @@ class ServiceUIViewSet(NautobotUIViewSet):  # 3.0 TODO: remove, unused BulkImpor
             object_detail.ObjectFieldsPanel(
                 section=SectionChoices.LEFT_HALF,
                 weight=100,
-                fields=[
-                    "name",
-                    "parent",
-                    "protocol",
-                    "port_list",
-                    "ip_addresses",
-                    "description",
-                ],
-                value_transforms={"ip_addresses": [to_list]},
+                fields=["name", "parent", "protocol", "port_list", "description"],
+            ),
+            object_detail.ObjectsTablePanel(
+                weight=200,
+                section=SectionChoices.RIGHT_HALF,
+                context_table_key="ip_addresses_table",
```

Comment:
Instead of manually constructing the table in `get_extra_context` and using a `context_table_key`, it should be possible (and simpler) to use a `table_class` and `table_filter` here, probably:

```suggestion
                table_class=tables.IPAddressTable,
                table_filter="services",
                add_button_route=None,
```